packages/core/useElementVisibility/index.ts

```diff
@@ -27,25 +27,26 @@ export function useElementVisibility(
       return
 
     const document = window.document
-    if (!element.value) {
+    if (!unref(element)) {
       elementIsVisible.value = false
     }
     else {
-      const rect = element.value.getBoundingClientRect()
-
-      elementIsVisible.value = (
-        rect.top <= (window.innerHeight || document.documentElement.clientHeight)
-          && rect.left <= (window.innerWidth || document.documentElement.clientWidth)
-          && rect.bottom >= 0
-          && rect.right >= 0
-      )
+      const rect = unref(element)?.getBoundingClientRect()
+      if (rect) {
```

Comment:
Why we need this conditions,It‘s seem to always return true.

Reply:
Yes, don't need